components/side-drawer/src/vwc-side-drawer.scss

```
@@ -21,6 +21,8 @@ $inline-size: var(#{variables.$side-drawer-inline-size}, 280px);
 .side-drawer {
 	position: fixed;
 	z-index: var(#{variables.$side-drawer-z-index}, #{$z-index-default});
+	top: 0;
```

```suggestion
	top: auto;
```
this will fix the overlapping top bar and the side drawer